packages/db/src/migration-template.ts

```diff
@@ -0,0 +1,9 @@
+import * as Knex from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => {
```

Comment:
Is this specifically for this ticket or all migration stuff? having an up and down

Reply:
Since I needed to add a new migration for this, I noticed that the template and knexfile weren't carried over to the db package. So this will allow devs to create new migrations going forward